app/controllers/cascade_controller.rb

```diff
@@ -9,7 +9,7 @@ def import
       @sign = FetchDeviceDataJob.perform_now(url: "#{params[:cascade_url]}/slideshow.json")
       @sign.slides.each{|slide| slide.owners << current_user }
       @sign.users << current_user
-      @sign.slides.each(&:take_screenshot)
+      @sign.slides.each {|slide| slide.take_screenshot(true) }
```

Comment:
I usually shy away from adding mysterious booleans as method parameters.  In order for someone to know what this means, they have to look the implimentation.  A better way would be to pass in a hash of args. `take_screenshot(validate_size: false)` maybe...